client/app/components/admin/Layout.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+import { PageHeader } from '@/components/PageHeader';
+
+export default function Layout({ activeTab, children }) {
+  return (
+    <div className="container">
+      <PageHeader title="Admin" />
+
+      <div className="bg-white tiled">
+        <ul className="tab-nav">
```

Comment:
Can't we use Ant's tabs here?

Reply:
If tab header can be a link and have an empty body - yes. I'll check